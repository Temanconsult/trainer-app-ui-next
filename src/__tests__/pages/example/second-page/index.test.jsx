import React from 'react'
import { render } from '@testing-library/react'
import App from '../../../../pages/example/second-page'

describe('Pages:SecondPage', () => {
  it('renders a second example page', () => {
    const { asFragment } = render(<App />)
    expect(asFragment()).toMatchSnapshot()
  })
})
