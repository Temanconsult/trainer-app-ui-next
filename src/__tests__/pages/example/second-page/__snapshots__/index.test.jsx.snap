// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages:SecondPage renders a second example page 1`] = `
<DocumentFragment>
  <div
    class="jsx-3583494339 text-center"
  >
    <main
      class="jsx-3583494339"
    >
      <p
        class="jsx-3583494339"
      >
        This is another page at a different URL.
      </p>
      <p
        class="jsx-3583494339"
      >
        Return to theÂ 
        <a
          href="/"
        >
          homepage
        </a>
      </p>
    </main>
  </div>
</DocumentFragment>
`;
